Terminal

cees91 opened the pull request.

cees91 committed:
Added a terminal which keeps running until user quits (enters q)

cees91 committed:
Added some initial 'routes' for the console app.

cees91 committed:
Merge remote-tracking branch 'remotes/origin/develop' into terminal

# Conflicts:
#	Hotel/src/hotel/App.java

JorisdeJong-cap commented on Hotel/src/hotel/Terminal.java:
```diff
@@ -0,0 +1,67 @@
+        Scanner terminalInput = new Scanner(System.in);
+        System.out.println(currentScreen);
+        String newInput = terminalInput.nextLine();
+        String input = previousInput != null? previousInput +","+ newInput:  newInput;
```
I find these kind of if statements hard to read, especially for newer programmers, we should take them into account.

JorisdeJong-cap committed:
Delete beers.java

Should not be in this project, we are not drinking beers, this is a serious hotel.

JorisdeJong-cap reviewed (approved):
the only thing is the shortened if statement, can be hard to read, especially for new programmers.
